ui/src/views/Hooks/ListHooks/index.jsx

```diff
@@ -135,16 +136,19 @@ export default class ListHooks extends Component {
         }>
         {!hookGroups && loading && <Spinner loading />}
         <ErrorPanel fixed error={error} />
-        {hookGroups && (
-          <TreeView
-            {...{ expanded, selected }}
-            onNodeToggle={handleState('expanded')}
-            onNodeSelect={handleState('selected')}
-            defaultCollapseIcon={<ArrowDropDownIcon />}
-            defaultExpandIcon={<ArrowRightIcon />}>
-            {tree}
-          </TreeView>
-        )}
+        {!loading &&
+          (tree.length ? (
+            <TreeView
+              {...{ expanded, selected }}
+              onNodeToggle={handleState('expanded')}
+              onNodeSelect={handleState('selected')}
+              defaultCollapseIcon={<ArrowDropDownIcon />}
+              defaultExpandIcon={<ArrowRightIcon />}>
+              {tree}
+            </TreeView>
+          ) : (
+            <Typography variant="subtitle1">No hooks are defined</Typography>
```

Comment:
I think this situation, there's nothing the user can do to fix it so this may not be that helpful of an error message. :) I'd show a generic error message (you can see if there are already some in helper files) like 'Something went wrong - please file a bug if the problem persists'.

Reply:
Hmm, It is something that existed as behavior a prior my changes, I just accidentally ate it in one of commits, so this change is more restoration to status quo... also there is a bug to change the text to "No hooks are defined" if hooks view is empty.

Reply:
Ok, I'd definitely switch to fetching directly from upstream (taskcluster repo) and rebasing so you keep branch clean, rather than merging. We can leave this for now, but fwiw, its definitely ok to change existing code if its not clear or not useful anymore :)

Reply:
I think you're going to need to rebase and drop any of the merge commits that don't belong here so you can get all tests to pass.

Reply:
@cerkiewny Hi, just following up with this in case you missed my reply :)

Reply:
I haven't missed it just had little time for TC last week and spent it trying to set up my k8s for it, I will probably follow it up today/tomorrow.

Reply:
so, should I remove this? (It is btw in case of empty task cluster hooks not in case of errors) I fixed all the other things I think.

Reply:
Well, wouldn't an empty taskcluster hook be an application error? It's not based on something the user did "wrong" so that message that is shown to the user doesn't seem very helpful in my opinion. If I saw that when trying to access menu items, I'd think that part of the app is broken. 

But you can just leave it for now so we can get this landed. It seems the service-github task is failing. @petemoore do you know if this is an intermittent failure?

Reply:
I thought it is meant for clean installation of taskcluster, if you dev init/ and start the backend, if there are no hooks defined yet, basically the message would show.